githubmarkdownui/job.py

```diff
@@ -0,0 +1,27 @@
+from typing import List, Optional
+
+from githubmarkdownui.emoji import Emoji
+
+
+class Job:
```

Comment:
You might wanna consider [dataclasses](https://docs.python.org/3/library/dataclasses.html) and [dataclasses-json](https://pypi.org/project/dataclasses-json/) to save some boilerplate codes

Reply:
Thanks, dataclasses let me remove the `__init__` in this class and the list classes